app/models/content_item.rb

```diff
@@ -82,6 +82,25 @@ def as_json(options = nil)
     end
   end
 
+  # We store the description in a hash because Publishing API can send through
+  # multiple content types.
+  def description=(value)
+    super("value" => value)
+  end
+
+  def description
+    description = super
+
+    if description.is_a?(Hash)
+      description.fetch("value")
+    else
+      # This is here to ensure backwards compatibility during data migration:
+      # db/migrate/20151130111755_description_value_hash.rb
+      # It can be removed afterwards.
```

Comment:
The migration itself is running in the context of the new code (because it's a new `rake` process) but the web workers aren't restarted until after the migration has finished.  That's more of a problem with relational DBs because fixed schemas, but I dunno how mongo(id) would react.